library/CMakeLists.txt

```diff
@@ -111,8 +111,6 @@ if(NOT BUILD_SHARED_LIBS)
 	endif()
 endif()
 
-include("${CMAKE_CURRENT_SOURCE_DIR}/../cmake/colour.cmake")
```

Comment:
how come this got taken out?

Reply:
It already exists in another file, so don't need to include it here anymore (as the main CMakeLists.txt doesn't use it)